tox.ini

```
@@ -21,7 +21,7 @@ commands = {posargs}
 [testenv:pytest]
 deps =
   ansible-core
-  pytest
+  pytest==8.1.0
```

```suggestion
  pytest==8.1.1
```
Looks like 8.1.0 was yanked.

---

It's probably fine since zuul is pulling from repo mirrors (that I still haven't been able to find the source of) but I'd prefer to avoid any possible issues with the version that led to it being yanked.